### PR TITLE
Single session for multiple group chats

### PR DESCRIPTION
**Pull request by eugenia0804**
Add in helper.py and util.py.
Run helper.py to get fetched data from all group names in .xlsx
Change fetch_batch_size & fetch_limit to get more messages

### CODE REVIEW

File: helper.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+# Authenticate and store the session
+client = authenticate()
+
+import subprocess
+import re
+import shutil
+import os
+
+# List of group_names
+group_names = read_group_names("MASTER_NAWA_GAZA_Sources_2023_data_collecting.xlsx")
```
Comment (huslage) [inline helper.py:38]: Could this be passed in as an argument on the CLI? What are the columns required in it?

**Review by huslage — CHANGES_REQUESTED** [review]

File: helper.py (hunk 1 above)
Comment (eugenia0804) [reply helper.py:38]: Only the column Source ID is needed (urls with group names).

Comment (huslage) [reply helper.py:38]: So could we just do a list of channel names in a text file instead of an XLS?

Comment (eugenia0804) [reply helper.py:38]: Yep

**Pull request closed without merging**